shardingsphere-infra/shardingsphere-infra-binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/table/TablesContextTest.java

```diff
@@ -39,21 +42,21 @@
 import static org.mockito.Mockito.when;
 
 public final class TablesContextTest {
-    
+
     @Test
     public void assertGetTableNames() {
         TablesContext tablesContext = new TablesContext(Arrays.asList(createTableSegment("table_1", "tbl_1"), createTableSegment("table_2", "tbl_2")));
         assertThat(tablesContext.getTableNames(), is(Sets.newHashSet("table_1", "table_2")));
     }
-    
```

Comment:
@setamv Please keep indents consistent with the previous one.

Reply:
ok